### ISSUE TITLE
VisUI 1.2.0 and VisUI 1.2.1-SNAPSHOT compatibility "problems"

### ISSUE THREAD

**Issue by francogp** [kotcrab/vis-editor#212]
When I use libGDX 1.9.4 I get:

```
VisUI: Warning, using invalid libGDX version for VisUI 1.2.0.
You are using libGDX 1.9.4 but you need 1.9.3. This may cause unexpected problems and runtime exceptions.
```

tested with VisUI 1.2.0 and VisUI 1.2.1-SNAPSHOT

**Comment by kotcrab** [comment]
Oops forgot to update target gdx version but nothing to worry, it will work. Use `VisUI.setSkipGdxVersionCheck(true) to suppress this message.

**Comment by kotcrab** [comment]
I released 1.2.1 so you can just update.
